src/system/SystemPacketBuffer.cpp

```diff
@@ -355,6 +357,29 @@ PacketBuffer * PacketBuffer::Consume(uint16_t aConsumeLength)
     return lPacket;
 }
 
+CHIP_ERROR PacketBuffer::Read(uint8_t * aDestination, size_t aLength)
```

Comment:
Is there a reason not to make this function const?  Seems like we could.

Reply:
Yes. I forgot because I was also considering whether this should be a `ReadAndConsume()`, or an optional destination buffer on `Consume()`.